app/cdap/components/ConnectionList/index.tsx

```diff
@@ -55,8 +57,7 @@ export default function ConnectionList() {
   const queryParams = new URLSearchParams(loc.search);
   const pathFromUrl = queryParams.get('path') || '/';
   const [loading, setLoading] = useState(true);
-  const [isErrorOnNoWorkspace, setIsErrorOnNoWorkSpace] = useState<boolean>(false);
-
+  const [toaster, setToaster] = useState<ISnackbar>(snackbarDefaultValues);
```

Comment:
why not turn this into a custom state hook that contains the snackbar default state - also why change the naming of the snackbar in this situation? Why is it the snackbar component but this is [toaster, setToaster]? 

This should be `const [snackbarIsOpen, setSnackbar] = useSnackbar();` and there should be a useSnackbar custom hook to accompany it.